packages/donor/src/components/basic/Input/Input.tsx

```diff
@@ -9,70 +8,48 @@ export enum InputVariant {
   outlined = "outlined",
 }
 
-type InputProps = {
+export type InputProps = {
   id?: string;
   label?: string;
   onChangeText: (newValue: string) => void;
   value?: any;
   type?: string;
-  isDisabled?: boolean;
-  placeholder?: string;
   className?: string;
-  mainIcon?: any;
   errorMessage?: string;
-  actionIcon?: any;
   /** Standard - MUI design, Filled - With Background, or Outlined - With borders */
   variant?: InputVariant;
-  isFullWidth?: boolean;
-  onKeyPress?: (ev: any) => void;
+  onSubmit?: () => void;
 };
 
 export default function Input({
   id,
   label,
-  placeholder,
   type = "text",
   value,
   onChangeText,
   className,
-  mainIcon,
-  actionIcon,
-  isDisabled,
   errorMessage,
   variant = InputVariant.standard,
-  isFullWidth = false,
-  onKeyPress = (ev) => undefined,
+  onSubmit,
 }: InputProps) {
-  let adornments = {
```

Comment:
Why are we removing the adronments?

Reply:
No one uses it in V1. If we ever need it we can add it. Don't want to have הכנה למזגן in basic components